tc/core/polyhedral/scop.h

```diff
@@ -437,6 +438,13 @@ struct Scop {
       isl::schedule_constraints constraints,
       const SchedulerOptionsView& schedulerOptions);
 
+ public:
+  // Do the simplest possible dependence analysis.
+  // Compute all RAW, WAR, and WAW dependences, and save them in dependences.
+  void computeAllDependences();
+  // Save the RAW, WAR, and WAW dependences
```

Comment:
Saying "save" sounds like a function, but it's a member.  Let's just say "RAW, WAR and WAW dependences"